exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -53,6 +53,11 @@ def initialize(endpoint: config_opt('OTEL_EXPORTER_OTLP_TRACES_ENDPOINT', 'OTEL_
           raise ArgumentError, "invalid url for OTLP::Exporter #{endpoint}" if invalid_url?(endpoint)
           raise ArgumentError, "unsupported compression key #{compression}" unless compression.nil? || %w[gzip none].include?(compression)
 
+          # TODO: add OTEL_EXPORTER_OTLP_METRICS_PROTOCOL protocol env var support when metrics api/sdk is added.
+          # This may need to be moved to sdk/configurator.rb when grpc and http/json options are added, if added as unique gems.
```

Comment:
Is it up in the air at all about whether to use separate gems? I'd argue for putting this in the SDK now if that is the plan. Then the code for selecting which gem's exporter to use could be included already as well -- I assume it would be supported to have multiple exporter gems included as deps, so `protocol` in the SDK would be used to select which to use, or error out if the user specifies to use one but hasn't included its gem.

Reply:
sorry didnt see this comment. sure, yea i mean, i dont have strong opinions but i wasn't sure if we could 1.0 an exporter gem without explicitly supporting this option in the gem itself.

Reply:
We can 1.0 this gem without supporting the option. We want to stick the gRPC support in a separate gem, selected by this env var in the SDK. I.e. the code below belongs in the Configurator.